src/custom/pages/Profile/index.tsx

```diff
@@ -228,7 +229,7 @@ export default function Profile() {
 
             <AddToMetamask currency={COW[chainId || 1] as Currency | undefined} />
 
-            <ExtLink href={'#'}>Buy COW ↗</ExtLink>
+            <Link to={`/swap?outputCurrency=${COW_CONTRACT_ADDRESS[chainId || 1]}`}>Buy COW ↗</Link>
```

Comment:
you'll see these comments a lot in your waterfalls, so i'll just write this one and you change all the subsequent appearances of this. Please use SupportedChainId.MAINNET